File: Com.Fubei.Api.OpenApi/Net/HttpClient/HttpClientFactory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Com.Illuminati.Galileo.Biz.MerchantApi;
using Com.Illuminati.Galileo.Net.HttpClient.Impl;

namespace Com.Illuminati.Galileo.Net.HttpClient
{
    public class HttpClientFactory
    {
        public static IHttpRequest<OpenApiResponse> HttpRequestForOpenApi { get; } = new FubeiHttpRequestAdapter<RestSharpRequestImpl, OpenApiResponse>();
    }
}

File: Com.Fubei.OpenApi.Vb/Net/HttpClient/HttpClientFactory.vb
Imports Com.Fubei.Api.OpenApi.Biz
Imports Com.Illuminati.Galileo.Net.HttpClient
Imports Com.Illuminati.Galileo.Net.HttpClient.Impl

Namespace Net.HttpClient
    Public Class HttpClientFactory
        Public Shared ReadOnly Property HttpRequestForOpenApi As IHttpRequest(Of OpenApiResponse) = New FubeiHttpRequestAdapter(Of RestSharpRequestImpl, OpenApiResponse)()
    End Class
End Namespace
